test/query.js

```diff
@@ -990,7 +992,104 @@ describe('then', function() {
     User.filter({}).then(function() {
         done();
     });
+  });
+  it('should return a promise', function(done) {
+    var name = util.s8();
+
+    var Query = thinky.Query;
+    var r = thinky.r;
+    var User = thinky.createModel(modelNames[0], {id: String}, {init: false});
+
+    User.filter({}).then(function() {
+        // no-op
```

Comment:
Why don't you call `done` here instead of chaining `spread`?

```
User.filter({}).then(function(result) {
  assert(Array.isArray(result));
  done();
}).catch(done)
```